apps/institution-service/src/controllers/institutionController.test.ts

```diff
@@ -760,6 +764,60 @@ describe("institutionController", () => {
 
       expect(inactiveInstitutionFound).toBeFalsy();
     });
+
+    it("uses default sort order if sortBy is not passed in to request", async () => {
+      const req = buildInstitutionRequest({
+        aggregatorName: ["testExampleA"],
+      });
+
+      const res = {
+        json: jest.fn(),
+        status: jest.fn().mockReturnThis(),
+      } as unknown as Response;
+
+      const findAndCountAllSpyCustom = jest.spyOn(
+        Institution,
+        "findAndCountAll",
+      );
+
+      await getPaginatedInstitutions(req, res);
+
+      expect(findAndCountAllSpyCustom).toHaveBeenCalledWith(
```

Comment:
Testing that findAndCountAll is called with a certain order is testing an implementation detail.

The sequelize api could change to be something other than order and these tests would still pass.

If you check the actual results of the function and test that they are being sorted correctly, then we have the confidence we need.